cmd/zfs/zfs_main.c

```diff
@@ -4449,10 +4449,16 @@ static boolean_t
 zfs_do_send_exclude(zfs_handle_t *zhp, void *context)
 {
 	zfs_send_exclude_arg_t *exclude = context;
-	const char *name = zfs_get_name(zhp);
+	const char *poolname = zfs_get_pool_name(zhp);
+	const char *fullname = zfs_get_name(zhp);
 	size_t indx;
+	const char *name;
+
+	name = fullname + strlen(poolname);
+	if (name && *name == '/')
```

Comment:
redundant: `name` always holds

Reply:
Grr that was supposed to have been changed.  Thanks. grumble.  (That's why I converted this to a draft.)